rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section19/rule_19_31.json

```diff
@@ -0,0 +1,168 @@
+{
+    "rule-19-31-a": {
+        "Section": 19,
+        "Rule": 31,
+        "Test": "a",
+        "test_description": "Project contains a computer room served by a packaged terminal heat pump system. HVAC fans correctly operate continuously during unoccupied hours. ",
+        "expected_rule_outcome": "pass",
+        "standard": {
+            "rule_id": "19-31",
+            "ruleset_reference": "Table G3.1 Section 4 Exception #3",
+            "rule_description": "HVAC fans shall remain on during unoccupied hours in systems primarily serving computer rooms in the proposed design.",
+            "applicable_rmr": "Proposed RMR",
+            "rule_assertion": "=",
+            "comparison_value": "Expected Value",
+            "primary_rule": "Yes",
+            "schema_version": "0.0.23"
+        },
+        "rmr_transformations": {
+            "proposed": {
+                "id": "ASHRAE229 1",
+                "ruleset_model_instances": [
+                    {
+                        "id": "RMD 1",
+                        "buildings": [
+                            {
+                                "id": "Building 1",
+                                "building_open_schedule": "Required Building Schedule 1",
+                                "building_segments": [
+                                    {
+                                        "id": "Building Segment 1",
+                                        "zones": [
+                                            {
+                                                "id": "Thermal Zone 1",
+                                                "thermostat_cooling_setpoint_schedule": "Required Cooling Schedule 1",
+                                                "thermostat_heating_setpoint_schedule": "Required Heating Schedule 1",
+                                                "terminals": [
+                                                    {
+                                                        "id": "PTHP Terminal 1",
+                                                        "is_supply_ducted": false,
+                                                        "type": "CONSTANT_AIR_VOLUME",
+                                                        "served_by_heating_ventilating_air_conditioning_system": "PTHP 1"
+                                                    }
+                                                ],
+                                                "spaces": [
+                                                    {
+                                                        "id": "Space 1",
+                                                        "lighting_space_type": "COMPUTER_ROOM"
```

Comment:
Similar to Rule 19 28, 29, Could you add ```floor_area``` key here?

Reply:
Added